### PR TITLE
[zh-cn]: create the translation of `VirtualKeyboard.boundingRect` property

### PR DESCRIPTION
**Pull request by tianyeeT**
…perty

### CODE REVIEW

File: files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+slug: Web/API/VirtualKeyboard/boundingRect
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+{{domxref("VirtualKeyboard")}} 接口的 **`boundingRect`** 属性包含一个 {{domxref("DOMRect")}}，用于指示覆盖在网页上的屏幕虚拟键盘的位置和大小。
+
+当使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可防止浏览器自动调整视窗大小，使屏幕虚拟键盘覆盖在视窗上。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:12]:
```suggestion
当使用 {{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可防止浏览器自动调整视窗大小，使屏幕虚拟键盘覆盖在视窗上。
```

参见 https://github.com/mdn/translated-content/pull/22039/files#r1660418839

下同

Comment (tianyeeT) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:12]:
```suggestion
当使用 {{domxref("VirtualKeyboard_API", "Virtual Keyboard API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可防止浏览器自动调整视口大小，使屏幕虚拟键盘覆盖在视口上。
```

File: files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{domxref("VirtualKeyboard_API", "VirtualKeyboard API", "", "nocode")}}
+- [使用 VirtualKeyboard API 实现完全控制](https://developer.chrome.com/docs/web-platform/virtual-keyboard/)
```
Comment (tianyeeT) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:35]:
```suggestion
- [使用虚拟键盘 API 实现完全控制](https://developer.chrome.com/docs/web-platform/virtual-keyboard/)
```

File: files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+slug: Web/API/VirtualKeyboard/boundingRect
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+{{domxref("VirtualKeyboard")}} 接口的 **`boundingRect`** 属性包含一个 {{domxref("DOMRect")}}，用于指示覆盖在网页上的屏幕虚拟键盘的位置和大小。
+
+当使用 {{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可防止浏览器自动调整视窗大小，使屏幕虚拟键盘覆盖在视窗上。
```
Comment (tianyeeT) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:12]:
```suggestion
当使用 {{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可防止浏览器自动调整视口大小，使屏幕虚拟键盘覆盖在视口上。
```

File: files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+title: VirtualKeyboard：boundingRect 属性
+slug: Web/API/VirtualKeyboard/boundingRect
+l10n:
+  sourceCommit: 5cdb341c723de0edb273769555d9124266d9c851
+---
+
+{{APIRef("VirtualKeyboard API")}}{{SeeCompatTable}}{{securecontext_header}}
+
+{{domxref("VirtualKeyboard")}} 接口的 **`boundingRect`** 属性包含一个 {{domxref("DOMRect")}}，用于指示覆盖在网页上的屏幕虚拟键盘的位置和大小。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:10]:
```suggestion
{{domxref("VirtualKeyboard")}} 接口的 **`boundingRect`** 属性包含一个用于指示覆盖在网页上的屏幕虚拟键盘的位置和大小的 {{domxref("DOMRect")}}。
```

File: files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+当使用 {{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可防止浏览器自动调整视口大小，使屏幕虚拟键盘覆盖在视口上。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:12]:
```suggestion
当使用{{domxref("VirtualKeyboard_API", "虚拟键盘 API", "", "nocode")}} 时，设置 {{domxref("VirtualKeyboard.overlaysContent", "overlaysContent")}} 属性为 `true` 可通过使屏幕虚拟键盘覆盖在视口上，以防止浏览器自动调整视口大小。
```

File: files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+## 值
+
+一个 {{domxref("DOMRect")}}，指示视窗中虚拟键盘的位置和大小。
```
Comment (yin1999) [inline files/zh-cn/web/api/virtualkeyboard/boundingrect/index.md:16]:
```suggestion
一个指示视窗中虚拟键盘的位置和大小的 {{domxref("DOMRect")}}。
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]

**Pull request merged**